### PR TITLE
Fix false positives from ActiveJobPerformable

### PR DESCRIPTION
**Pull request by rzane**
The following code was incorrectly being recognized as an offense:
```ruby
class MyBusinessLogic
  class MyJob < ApplicationJob
    def perform
      MyBusinessLogic.call
    end
  end
end
```

This was happening because `Node#descendants` returns all of the nodes descendants (recursively), rather than the node's immediate children.

### CODE REVIEW

File: lib/rubocop/cop/betterment/active_job_performable.rb
Comment (rzane) [inline lib/rubocop/cop/betterment/active_job_performable.rb:44]: When a class contains a single method, `ClassNode#body` is a `DefNode`. Otherwise, `ClassNode#body` is a `BeginNode`.

File: spec/rubocop/cop/betterment/active_job_performable_spec.rb
Comment (rzane) [inline spec/rubocop/cop/betterment/active_job_performable_spec.rb:5]: I tried to convert these to use `expect_offense` to improve the usefulness the spec's failure message, but Rubocop tried to parse the URL in the offense and choked.

**Review by smudge — APPROVED** [review]
domain LGTM && platform LGTM!

**Comment by smudge** [comment]
@rzane though you may want to increment the version number if you want a new release out after this merges.

**Review by smudge — APPROVED** [review]
domain LGTM

**Comment by rzane** [comment]
@smudge, not sure what's going on, but I don't see a merge button. I'm guessing I don't have permission?

**Comment by aburgel** [comment]
@rzane @smudge This change breaks on empty classes:

```ruby
class MyClass
end
```

```
     NoMethodError:
       undefined method `begin_type?' for nil:NilClass
     # ./lib/rubocop/cop/betterment/active_job_performable.rb:39:in `possible_methods_within'
     # ./lib/rubocop/cop/betterment/active_job_performable.rb:35:in `has_perform_method?'
     # ./lib/rubocop/cop/betterment/active_job_performable.rb:27:in `on_class'
```
`nody.body` is nil in this case. `return [] if node.body.nil?` at the beginning of `possible_methods_within` will fix it.

**Comment by rzane** [comment]
Ah! Thank you @aburgel! I'll fix this right away!

**Pull request merged**
